merkle/rfc6962/rfc6962_test.go

```diff
@@ -196,5 +198,14 @@ func TestHashChildrenEquivToOld(t *testing.T) {
 			t.Errorf("%d different hashes: %x vs %x", i, oldHash, newHash)
 		}
 	}
+}
 
+// Tests that the rfc6962 log hashers were registered.
+func TestRegistrations(t *testing.T) {
```

Comment:
TODO adding other hashers to this test in follow-ups?

Reply:
These are the ones registered by the code being tested. Others should be tested in an appropriate place.